### ISSUE TITLE
CI tests failing - scrollable-region-focusable

### ISSUE THREAD

**Issue by davidwagner** [data-8/textbook#215]
The automatic CI tests ("Axe Checks") are currently failing.  Let's fix this, so that all tests pass and we can rely on them to make sure that revisions don't break accessibility.  The current error:

```
Testing http://localhost:8080/chapters/14/1/properties-of-the-mean/index.html ... please wait, this may take a minute.

  Violation of "scrollable-region-focusable" with 1 occurrences!
    Ensure elements that have scrollable content are accessible by keyboard. Correct invalid elements at:
     - #fOIqljKHwR > .myst-code.hover\:shadow-md.dark\:shadow-2xl > pre
    For details, see: https://dequeuniversity.com/rules/axe/4.11/scrollable-region-focusable
```

**Comment by pancakereport** [comment]
Yep! Fixing upstream later this month! https://github.com/jupyter-book/myst-theme/issues/743

(This means I can make the fix in one place and don't need to implement for every textbook/course notes repo that I help manage.)
